docs/formatV3.txt

```diff
@@ -341,6 +341,10 @@ Credit Card Expiration      0x1d        Text          N              [24]
 Credit Card Verif. Value    0x1e        Text          N              [24]
 Credit Card PIN             0x1f        Text          N              [24]
 QR Code                     0x20        Text          N              [25]
+TOTP Config                 0x21        1 byte        N              [29]
+TOTP Digits                 0x22        1 byte        N              [29]
```

Comment:
I think "TOTP Length" would be clearer, no?

Reply:
In the RFC, there's a section (5.1) defining "Digit" as the "number of digits in a HOTP value [...]".
Since we don't have such as section, I vote for clarity, that is, "TOTP Length".
If we really want to be pedantic, we can note that this corresponds to the term "Digits" in the aforementioned RFC.